examples/bluetooth/blufi/CMakeLists.txt

```diff
@@ -3,4 +3,4 @@
 cmake_minimum_required(VERSION 3.5)
 
 include($ENV{IDF_PATH}/tools/cmake/project.cmake)
-project(blufi_demo)
+project(blufi)
```

Comment:
Should a corresponding change be made even in examples/bluetooth/bluedroid/ble/blufi/Makefile  file ?